code/modules/projectiles/projectile.dm

```diff
@@ -836,6 +840,11 @@
 	var/obj/item/projectile/P = A
 	return istype(P) && P.is_reflectable
 
+//Bullet types that are blocked/deflected by power armor
+/proc/can_penetrate_power_armor(atom/A)
```

Comment:
See above. This is, frankly, a silly proc as well; it's hyper-specific and all it does is run an istype and var access. If you still want to keep `power_armor_blocks` (you shouldn't), this should be either:
1) A proc on power armour, like `can_penetrate(atom/A)`, called in `run_block`. This is the ideal option.
or
2) Inlined in `run_block`, since it's just an istype and variable access:
```
	var/blocked = FALSE
	if(istype(A, /obj/item/projectile))
		var/obj/item/projectile/P = A
		blocked = P.power_armor_blocks
	if(blocked && ...)
```
etc.